ffmpeg.h

```diff
@@ -25,6 +25,12 @@
 
 #endif
 
+/*
+ * AVPixelFormat is a newer release of avutil PixelFormat is the old name
+ */
+#ifndef AVPixelFormat
```

Comment:
I tested and it does work. The #define here is just a way of giving the enum the original name.  I was afraid the underlying libs changed but didn't seem to.